lib/task/import/csvImportTask.class.php

```diff
@@ -979,7 +998,19 @@ public function execute($arguments = array(), $options = array())
 
         // This will import only a single digital object;
         // if both a URI and path are provided, the former is preferred.
-        if ($uri = $self->rowStatusVars['digitalObjectURI'])
+
+        // note: Digital Objects should only be created here if they do not
+        // exist already.  If getDigitalObject() is null, we know they do not
+        // exist and therefore should be created.
+
+        // When this CSV importer is run and the --update parameter is set,
+        // the function 'updatePreparationLogic' (above) will be run.
+        // 'updatePreparationLogic' has been enhanced to compare the DO
+        // checksum in the DB against the checksum in the import CSV and will
+        // only remove the DO if the checksums differ.  If they are removed
+        // because the checksums are different, the following code will
+        // recreate them from the CSV file.
+        if (($uri = $self->rowStatusVars['digitalObjectURI']) && null === $self->object->getDigitalObject())
```

Comment:
I think the parentheses around `($uri = $self->rowStatusVars['digitalObjectURI'])` are unnecessary.